src/pages/Pools.tsx

```diff
@@ -214,18 +261,27 @@ function Pools(): ReactElement | null {
           <ReviewMigration
             onClose={(): void => {
               setCurrentModal(null)
-              setActiveMigration(null)
+              setActiveMigration({
+                poolName: STABLECOIN_POOL_NAME,
+                lpTokenBalance: Zero,
+              })
             }}
             onConfirm={async (): Promise<void> => {
               setCurrentModal("confirm")
               logEvent("migrate", {
-                pool: activeMigration,
+                pool: activeMigration.poolName,
               })
-              await approveAndMigrateUSD(usdUserShareData?.lpTokenBalance)
+              await approveAndMigrate(
+                activeMigration.poolName,
+                activeMigration.lpTokenBalance,
```

Comment:
Prefer to read lptokenbalance from `getPropsForPool(poolName)` or maybe even just read it directly from `usePoolData` in the `useMigrate` hook